src/vos/vos_pool.c

```diff
@@ -789,36 +789,39 @@ vos_pool_open_metrics(const char *path, uuid_t uuid, unsigned int flags, void *m
 	struct d_uuid		 ukey;
 	PMEMobjpool		*ph;
 	int			 rc, enabled = 1;
+	bool			 skip_uuid_check = flags & VOS_POF_SKIP_UUID_CHECK;
```

Comment:
I don't quite see how a pool can be opened successfully without knowing pool UUID, it needs to get the SPDK blob ID from pool UUID and target ID.

What 'uuid' will be provided in this case? An empty UUID? Could you elaborate the requirement a bit?

Reply:
The primary purpose is for the use of the daos debug tool which is part of the catastrophic recovery milestone. It is meant to connect to a single vos file and be able to navigate the vos tree, inspecting and altering it. While the vos file is typically within a path that contains the pool uuid, that may not always be the case (i.e. a user sends a vos file in email for support). It was discussed in the catastrophic recovery WG meeting that being able to open a vos file while not knowing the pool uuid is desirable for the daos debug tool.

Yes, an empty UUID will be provided in this case.